data_structures/src/superblock.rs

```diff
@@ -1432,7 +1435,7 @@ mod tests {
         // Receive three superblock votes for index 1
         // Since the signing_committee_size is 2, one of the votes will not be valid
         let mut v1 = SuperBlockVote::new_unsigned(sb2_hash, 1);
-        v1.secp256k1_signature.public_key = p1;
+        v1.secp256k1_signature.public_key = p3;
```

Comment:
why you change the pk?

Reply:
because with the index being taken into account, the public key that is valid is p3 and not p1.